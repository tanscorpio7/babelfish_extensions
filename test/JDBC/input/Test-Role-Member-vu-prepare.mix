CREATE LOGIN test_role_member_l1 WITH PASSWORD = '12345678';
GO

CREATE ROLE test_role_member_r1;
GO

CREATE USER test_role_member_u1 FOR LOGIN test_role_member_l1;
GO

CREATE TABLE test_role_member_t1(a INT);
GO

GRANT SELECT ON OBJECT::test_role_member_t1 to test_role_member_r1;
GO

-- tsql		user=test_role_member_l1		password=12345678
SELECT * FROM test_role_member_t1;
GO

-- tsql
ALTER ROLE test_role_member_r1 ADD MEMBER test_role_member_u1;
GO

-- tsql		user=test_role_member_l1		password=12345678
SELECT * FROM test_role_member_t1;
GO

-- tsql
use master
GO
