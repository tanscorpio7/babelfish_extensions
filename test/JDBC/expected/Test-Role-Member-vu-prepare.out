CREATE LOGIN test_role_member_l1 WITH PASSWORD = '12345678';
GO

CREATE ROLE test_role_member_r1;
GO

CREATE USER test_role_member_u1 FOR LOGIN test_role_member_l1;
GO

CREATE TABLE test_role_member_t1(a INT);
GO

GRANT SELECT ON OBJECT::test_role_member_t1 to test_role_member_r1;
GO

-- tsql		user=test_role_member_l1		password=12345678
SELECT * FROM test_role_member_t1;
GO
~~ERROR (Code: 33557097)~~

~~ERROR (Message: permission denied for table test_role_member_t1)~~


-- tsql
ALTER ROLE test_role_member_r1 ADD MEMBER test_role_member_u1;
GO

-- tsql		user=test_role_member_l1		password=12345678
SELECT * FROM test_role_member_t1;
GO
~~START~~
int
~~END~~


-- tsql
use master
GO
